stroom-connectors/src/main/java/stroom/connectors/ConnectorPropertiesEmptyImpl.java

```diff
@@ -0,0 +1,31 @@
+package stroom.connectors;
```

Comment:
This implementation can be used for testing, where a full Stroom Property Service may not be available.